Source/inv.cpp

```diff
@@ -804,7 +804,7 @@ bool AutoEquip(int playerNumber, const ItemStruct &item, bool persistItem)
 	}
 
 	for (int bodyLocation = INVLOC_HEAD; bodyLocation < NUM_INVLOC; bodyLocation++) {
-		if (AutoEquip(playerNumber, item, bodyLocation, persistItem)) {
+		if (AutoEquip(playerNumber, item, (inv_body_loc)bodyLocation, persistItem)) {

```

Comment:
Instead of casting, can't we just change the loop variable from `int` to `inv_body_loc`?

Reply:
unfortunately not, enum dosent understand ++